README.ja.md

```diff
@@ -0,0 +1,164 @@
+<head>
+  <meta name="Momento .NET Client Library Documentation" content=".NET client software development kit for Momento Serverless Cache">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-beta.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
```

Comment:
I think that copying this stuff is the right thing to do for now but we should create a ticket on the project board about extending the template tool to support READMEs for JP.  Otherwise we'll have to be careful to go update all of this header stuff in all of the repos whenever we change the template in the future.

Reply:
> @bruuuuuuuce do you know why @poppoerika 's commit is "not semantic"?

this pr only has 1 commit, which is `add Japanaese README`, which is not semantic. I think if you push another empty commit, it will pass because it will take the pr title, which is semantic

Reply:
ahhh, I was only looking at the PR title, didn't notice that the commit message had a different subject.